### DISCUSSION TITLE
Jio Phone F220B Install OmniSD , Enable Hotspot etc

### DISCUSSION THREAD

**Discussion by fawazahmed0** [fawazahmed0/tiger#9] [Show and tell]
All files are available at [here](https://github.com/fawazahmed0/tiger/releases/download/v2/JioPhone.zip)
**Downgrade phone os to 003-01-26-110119**
Ref:
[link1](https://www.techforus.in/2020/02/install-hotspot-in-jio-phone-f220b.html)
[youtube](https://www.youtube.com/watch?v=KLMeXSD8uk8)

You can check the OS details
Settings -> Device-> Device Information -> More Information -> See Build Number

**Steps to Downgrade:**
1.Install Qualcomm USB Driver (QDLoader HS-USB Driver_64bit_Setup.exe) with default options & QPST Flash tool (I used v2.7.480)

2. Reboot

3.Run QFIL as admin (from start menu)
4.Extract F220B-003-01-26-110119Firmware.zip
5.Select Meta Build option in QFIL
6. Click Load Content -> Extracted Firmware folder -> contents.xml
7. Switch off phone, press * & insert usb cable, you will listen sound & it will show COM port status in QFIL
8. Click Download Content, & wait till it shows Download success
9. Remove phone & reinsert batter & start phone.

**Steps to install omnisd:**
Ref: [youtube](https://www.youtube.com/watch?v=kxJP2cH9LNA)
1. Extract LYF_F220B_OMNISD.7z
2. QFIL (as admin) , flat build
3. Programmer path: select omnisd/prog_emmc_firehose_8909_ddr.mbn
4. click Load XML & select the rawprogram**.xml & close other dialog
5. Switch off phone, press * & insert usb cable, you will listen sound
6. Click download & wait till it shows Download success
9. Remove phone & reinsert batter & start phone.

**Install Apps from omnisd:**
Paste appname.zip file at SDcard/downloads or internal/downloads
Go to omnisd app & install the app

Install Enable Stuff by Vivek.zip & Settings by Mobile and Computer Tech.zip using above i.e using omnisd
And open both apps onces after installation
(You can enable hotspot etc from new settings app)

**Disable Auto update:**
Ref:
[link1](https://www.techforus.in/2020/03/lyf-f220b-block-auto-update-2020.html)
[youtube](https://www.youtube.com/watch?v=M01vImHaoyo)

Extract ADB & Root.rar
Open QFIL , Flat build, 
Programming path: adb&root/prog_emmc_firehose_8909_ddr.mbn
Load xml & select default files
click Download & wait till success
reboot
oPen Enable Stuff by Vivek & exit
Enable Jio Phone setting \ Device \ Developer \ debugger \ ADB and DEV tools
(Install adb driver if not already using https://bit.ly/39uN0fO)
Run following adb command:

```
adb shell
mount -o remount,rw /system
echo '127.0.0.1 fota.mst.kai.jiophone.net' >> /system/etc/hosts
echo '127.0.0.1 jioads.akamaized.net' >> /system/etc/hosts
echo '127.0.0.1 ssp.kaiads.com' >> /system/etc/hosts
sync
mount -o remount,ro /system
exit
adb reboot

adb shell
cd /data/fota/downloaded
rm update.zip
cd /data/fota
rmdir downloaded
exit
adb reboot
```

**Comment by Entersjkhdfkjdhfksjf** [comment]
This is a bit late, but do you have something similar for the F30C?
